images/installer-openstack/root/usr/local/bin/entrypoint_casl

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source /opt/app-root/src/.config/openstack/openrc.sh
```

Comment:
we need to look for an approach that's a bit more flexible than this - alternatively having the option to map file `xyz.sh` to `openrc.sh`

Reply:
NVM on this one - we should be good as-is since we can just loop-back mount the specific RC file, mapping it to the `openrc.sh` file